src/stdatamodels/jwst/datamodels/schemas/core.schema.yaml

```
@@ -1075,6 +1075,46 @@ properties:
             type: integer
             fits_keyword: DETYSIZ
             blend_table: True
+          multistripe_reads1:
+            title: Num. substripe rows read starting at SUBSTRT1
+            type: number
```

```suggestion
            title: Number of substripe rows read starting at SUBSTRT1
            type: number
```
Will this ever be a non-integer value? If not, any reason to not make it an integer?